src/style.css

```diff
@@ -0,0 +1,85 @@
+* {
+  margin-top: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
```

Comment:
Please, check the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)